packages/containers/src/ComponentForm/ComponentForm.sagas.js

```diff
@@ -75,13 +75,18 @@ function* onFormSubmit(componentId, submitURL, action) {
 	}
 	const { response, data } = yield call(cmf.sagas.http.post, submitURL, action.properties);
 	if (!response.ok) {
-		return;
+		yield put({
```

Comment:
I don't see unit tests for `onFormSubmit` generator, any particular reason ?

Reply:
same `put`, DRY, just switch type value regarding the response status